modules/unauthenticated-users-cluster-role-binding-con.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * authentication/using-rbac.adoc
+// * post_installation_configuration/preparing-for-users.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="unauthenticated-users-cluster-role-bindings-concept_{context}"]
+= `ClusterRoleBindings` for unauthenticated groups
+
+{product-title} {product-version} does not allow unauthenticated groups access to `ClusterRoleBindings` for security.
+
+In previous {product-title} versions unauthenticated groups were included in accessing `ClusterRoleBindings`. Clusters depolyed with previous {product-title} versions will still behave this way.
+
+In specific use cases unauthenticated users may need access to ClusterRoleBindings. For example, the `system:webhook` role allows unauthenticated users to trigger builds from external systems like GitHub, GitLab, and Bitbucket, which is important to BuildConfigs processes.
+
+Cluster administrators can add unauthenticated users to the following `ClusterRoleBindings`:
+
+. `system:scope-impersonation`
+. `system:webhook`
+. `system:oauth-token-deleter`
+. `self-access-reviewer`
+
+[IMPORTANT]
+====
+Always verify compliance with your organization's security standards when modifying `RoleBindings`. Particularly when including unauthenticated access.
```

Comment:
The second sentence is a fragment. I'd reword to include the part about especially unauthenticated access into a single sentence.

Also write as just use "role bindings". Or should this be "cluster role bindings" here?

Reply:
Role bindings and cluster role bindings are different things.

Reply:
Right, but since you've been specifically talking about cluster role bindings throughout these modules, shouldn't we specifically say "cluster role bindings" here instead of just "role bindings"? I'd just at least confirm with an SME before leaving it as something different.